README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
The use of a semicolon is usually to separate statements, but there doesn't seem to be a statement before this semicolon. Is this semicolon needed?